forc-plugins/forc-client/src/cmd/submit.rs

```diff
@@ -2,9 +2,17 @@ use crate::NodeTarget;
 use devault::Devault;
 use std::path::PathBuf;
 
+forc::cli_examples! {
+    [ Submit a transaction from a json file => submit "./mint.json" => "Submission of tx or awaiting commit failed" ]
+    [ Submit a transaction from a json file and wait for confirmation => submit "./mint.json --await true" => "Submission of tx or awaiting commit failed" ]
+    [ Submit a transaction from a json file and get output in json => submit "./mint.json --tx-status-json true" => "Submission of tx or awaiting commit failed" ]
+    [ Submit a transaction from a json file to testnet => submit "./mint.json --testnet" => "Submission of tx or awaiting commit failed" ]
+    [ Submit a transaction from a json file to a local net => submit "./mint.json --target local" => "Submission of tx or awaiting commit failed" ]
```

Comment:
I'm not a fan of having 2 different targets namely `--testnet` and then `--target local`. Can this be simplified (possibly in a separate issue) to `--target testnet` and `--target local`?

Reply:
Sounds like a reasonable suggestion to me. Would you mind opening an issue for this @Braqzen ?

Reply:
Raised https://github.com/FuelLabs/sway/issues/5430